Makefile

```
@@ -182,3 +182,6 @@ setup-synocommunity: setup
 		-e "s|DISTRIBUTOR_URL\s*=.*|DISTRIBUTOR_URL = https://synocommunity.com|" \
 		-e "s|REPORT_URL\s*=.*|REPORT_URL = https://github.com/SynoCommunity/spksrc/issues|" \
 		local.mk
+
+docker:
+	docker run -it -v $$(pwd):/spksrc ghcr.io/synocommunity/spksrc /bin/bash
```

```suggestion
	docker run -it -v $$(pwd):/spksrc ghcr.io/synocommunity/spksrc
```

It already runs bash by default.

```sh
$ docker run -it -v $(pwd):/spksrc ghcr.io/synocommunity/spksrc
root@f717a14d2a7b:/spksrc# echo $SHELL
/bin/bash
```

---

fixed